carina-api/src/main/java/com/qaprosoft/carina/core/foundation/api/http/Headers.java

```diff
@@ -0,0 +1,8 @@
+package com.qaprosoft.carina.core.foundation.api.http;
+
+public interface Headers {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ci.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1214)